apps/designer/app/designer/shared/client-settings/settings.ts

```diff
@@ -0,0 +1,51 @@
+import * as values from "./values";
```

Comment:
is is currently unused, I built it before I realized we can't save theme setting into local storage, but I think we will have a plenty of use cases for local user settings in the local storage as an option to  user settings in the database